### PR TITLE
Adds CocoaPods Rome (Too great of a plugin to be missed off 🚀).

### PR DESCRIPTION
**Pull request by jcampbell05**
Such a great plugin and its a shame it was missed off. Luckily Orta pointed me to it before I made my own.

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -17,6 +17,14 @@
       "description": "Deploys dependencies for a CocoaPods project without needing to clone the repo (Similar to Bundler's `--deployment`)."
     },
     {
+      "gem": "cocoapods-rome",
+      "name": "Rome",
+      "author": "Boris Bügling",
+      "social_media_url": "https://twitter.com/neonichu",
+      "url": "https://github.com/neonichu/rome",
+      "description": "Makes it easy to build a list of frameworks (Similar to Carthage)."
```
Comment (orta) [inline plugins.json]: Any chance you can use the original description? "Rome makes it easy to build a list of frameworks for consumption outside of Xcode, e.g. for a Swift script."

**Comment by jcampbell05** [comment]
@orta  updated.

**Comment by orta** [comment]
Ace, will merge on green!

**Comment by jcampbell05** [comment]
@orta 🍏

**Comment by orta** [comment]
Thanks!

**Pull request merged**